packages/runtime/container-runtime/api-report/container-runtime.api.md

```diff
@@ -121,11 +121,11 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents
     // (undocumented)
     get connected(): boolean;
     // (undocumented)
-    createDataStore(pkg: string | string[], groupId?: string): Promise<IDataStore>;
+    createDataStore(pkg: string | string[], dataStoreGroupIdForSnapshotFetch?: string): Promise<IDataStore>;
     // @deprecated (undocumented)
     _createDataStoreWithProps(pkg: string | string[], props?: any, id?: string): Promise<IDataStore>;
     // (undocumented)
-    createDetachedDataStore(pkg: Readonly<string[]>, groupId?: string): IFluidDataStoreContextDetached;
+    createDetachedDataStore(pkg: Readonly<string[]>, dataStoreGroupIdForSnapshotFetch?: string): IFluidDataStoreContextDetached;
```

Comment:
I do not think a rename like that would help. If anything, I think it becomes even more confusing.
I'd personally keep the old name, but ensure that every place that introduces this concept has a link to proper documentation.

If we really want to put more descriptive name, then maybe loadingGroupId is a better term.
I'd definitely remove "dataStore" from the name as it's not adding any value (API is already on data store, and loading flow does not care where these IDs should up - on data stores or DDSs or somewhere else).

Reply:
Renamed it to "loadingGroupId" on the api.

Reply:
What's the reason for not using 'groupId' alone?